core/src/epicli/data/common/defaults/configuration/helm.yml

```diff
@@ -0,0 +1,4 @@
+kind: configuration/helm
+title: "Helm"
+name: default
+specification: {}
```

Comment:
He is a good place of path and for helm version? Then this should be easy to update and other components also have their versions defined there.